events/syscall/launch_suspicious_network_tool_on_host.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(LaunchSuspiciousNetworkToolOnHost)
+
+func LaunchSuspiciousNetworkToolOnHost(h events.Helper) error {
+
+    cmd := exec.Command("nmap", "-sn", "192.168.1.0/24")
```

Comment:
Can we first check if `nmap` is installed and give proper feedback to the user in case it is not?
Ie: 
```
nmap, err := exec.LookPath("nmap")
if err != nil {
  h.Log().Warnf("nmap is needed to launch this action")
  return err
}
...
```

Reply:
Yes @FedeDP this is a good informative change. Let me add it.